cookbooks/clearwater/recipes/openimscorehss.rb

```diff
@@ -32,53 +32,29 @@
 # under which the OpenSSL Project distributes the OpenSSL toolkit software,
 # as those licenses appear in the file LICENSE-OPENSSL.
 
-%w{subversion mysql-server libmysqlclient-dev libxml2 libxml2-dev bind9 flex bison libcurl4-openssl-dev openjdk-7-jre-headless openjdk-7-jdk ant}.each do |p|
-  package p do
-    action [:install]
-    options "--force-yes"
-  end
-end
-
-directory "/opt/OpenIMSCore" do
-end
-
-subversion "OpenIMSCore HSS" do
-  repository "http://svn.code.sf.net/p/openimscore/code/FHoSS/trunk"
-  destination "/opt/OpenIMSCore/FHoSS"
-  revision "1193"
-  action :sync
+# Tell apt about the repository server.
+cookbook_file "/etc/apt/sources.list.d/fhoss.list" do
```

Comment:
<https://github.com/opscode-cookbooks/apt#apt_repository> has support for doing this "right" so we should use it (especially as we already use the apt cookbook).